src/main/scala/circt/stage/ChiselStage.scala

```diff
@@ -211,6 +211,10 @@ object ChiselStage {
     *
     * @param gen a call-by-name Chisel module
     */
+  @deprecated(
+    "this exposes the internal Chisel circuit which was not supposed to be public---use either ChiselStage.convert or ChiselStage.emitCHIRRTL instead ",
+    "Chisel 5.0"
+  )
```

Comment:
```suggestion
  @deprecated(
    "this exposes the internal Chisel circuit which was not supposed to be public--use either ChiselStage.convert or ChiselStage.emitCHIRRTL instead",
    "Chisel 5.0"
  )
```
Minor nit on the message, feel free to ignore

Reply:
The use of a fixed-width em-dash was intentional as opposed to a fixed-width en-dash which would not be appropriate. I could use alternative punctuation, though. The space was unintentional. I'll remove that.